src/main/webapp/assets/css/global-custom.css

```diff
@@ -313,4 +313,10 @@ body{
     border-color: #CEA4F5;
     background-color: #CEA4F5;
     box-shadow: 0 4px 6px rgba(50, 50, 93, .11), 0 1px 3px rgba(0, 0, 0, .08);
+}
+
+.navbar-toggler-icon
+{
```

Comment:
Move this curly brace to upper line. 
`.navbar-toggler-icon {`

Reply:
https://github.com/sef-global/sef-site/pull/309